src/ui/public/test_harness/test_sharding/get_shard_num.js

```diff
@@ -0,0 +1,28 @@
+import murmurHash3 from 'murmurhash3js';
+
+// murmur hashes are 32bit unsigned integers
+const MAX_HASH = Math.pow(2, 32);
+
+/**
+*  Determine the shard number for a suite by hashing
+*  it's name and placing it based on the hash
+*
+*  @param {number} shardTotal - the total number of shards
+*  @param {string} suiteName - the suite name to hash
+*  @return {number} shardNum - 1-based shard number
+*/
+export function getShardNum(shardTotal, suiteName) {
```

Comment:
I know what you mean. But I think I'm homing in on the noun here, not the verb. My line of thought: What does the `num` represent here? It represents the shard within which the suite will be run. From the suite's point of view, what is this shard number? It's the shard it belongs to / is assigned to. We could also just rename this to `getShardForSuite` and refer to the returned number as the `suiteShard`.